index.d.ts

```diff
@@ -48,8 +48,8 @@ declare module "react-native-floating-action" {
     onPressMain?: (state: boolean) => any;
 
     // undocummented in the readme
-    actionsTextBackground: string;
-    actionsTextColor: string;
+    actionsTextBackground?: string;
```

Comment:
thanks for this... I completely forgot about this... I'm adding into the README and changing the name for `textBackground` and `textColor`